contrib/docker/datapusher/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:2.7-stretch
+
+WORKDIR /usr/src
+
+# Clone datapusher
+RUN git clone https://github.com/ckan/datapusher.git
```

Comment:
We should probably pin the datapusher version and clone a specific tag here. WDYT?